tuxemon/core/tools.py

```diff
@@ -385,7 +385,7 @@ def calc_dialog_rect(screen_rect):
     return rect
 
 
-def open_dialog(game, text, menu=None):
+def open_dialog(game, text, avatar=None, menu=None):
```

Comment:
in the future, please don't change the order of function parameters.  just add new ones to the end.  this could break existing code.  might be ok now, but its good practice to leave parameter order alone.

Reply:
Will keep in mind. It only required few changes, so like you said I'll leave it like this for now as it looks cleaner. In the future I shall add them at the end.